### DISCUSSION TITLE
CSS is not Rendering on Website

### DISCUSSION THREAD

**Discussion by Sukriti-sood** [vigneshshettyin/Flask-Generate-Certificate#156] [Q&A] [answered]
I wanted to make some UI Changes in the Website but any change in CSS is not rendering to the website ,I was doing changes in 
sb-admin-2.css

**Comment by vigneshshettyin** [comment]
> I wanted to make some UI Changes in the Website but any change in CSS is not rendering to the website ,I was doing changes in
> sb-admin-2.css

Hey, @Sukriti-sood let us know which page design you want to change. sb-admin-2.css is styling of login, register, forgot and dashboard pages.

**Reply by Sukriti-sood** [reply]
Login , Register , Forget-password  and Activation Code

**Reply by shades-7** [reply]
@Sukriti-sood please share your UI prototype of changes you want to make

**Comment by arpit456jain** [comment, marked as answer]
> I wanted to make some UI Changes in the Website but any change in CSS is not rendering to the website ,I was doing changes in
> sb-admin-2.css

@Sukriti-sood try (ctrl + shift + R) to reload
